micro-rdk/src/common/app_client.rs

```diff
@@ -323,6 +329,46 @@ impl AppClient {
         Ok((Box::new(cfg_response), datetime))
     }
 
+    pub async fn get_agent_config(&self) -> Result<Box<DeviceAgentConfigResponse>, AppClientError> {
+        let host_info = Some(HostInfo {
+            platform: "linux/amd64".to_string(),
+            distro: "esp32".to_string(),
+            tags: Default::default(),
+        });
+
+        let version_info = Some(VersionInfo {
+            agent_running: "none".to_string(),
+            ..Default::default()
+        });
```

Comment:
can't be `None` for sake of getting the information from the updated api

Reply:
is "none" the right string in this scenario then?

Reply:
discussed offline